### PR TITLE
Add AccountDomainNotFound error

### PR DESCRIPTION
**Pull request by wilsonianb**

### CODE REVIEW

**Comment by stevenzeiler** [comment]
:+1: LGTM

File: src/index.js [hunk 1]
```diff
@@ -57,11 +57,16 @@ export default class ValidatorDomainVerifier {
     const valPubKey = new ValPublicKey(validationPublicKey)
     const account_id = valPubKey.getAddress()
     ValidatorDomainVerifier._validateRippleAddress(account_id)
```
Comment (wilsonianb) [inline src/index.js:59]: I need to either stop throwing `InvalidRippleAccount` in this check or add that error as a valid verification error here:
https://github.com/ripple/validator-registry-api/blob/feature/verifications/api/models/verification.js#L15-L19

Comment (wilsonianb) [inline src/index.js:59]: I'm going to allow `InvalidRippleAccount` as a verifications error

**Comment by wilsonianb** [comment]
:+1:

**Pull request merged**